src/main/groovy/net/minecrell/gradle/licenser/tasks/LicenseTask.groovy

```diff
@@ -44,10 +44,14 @@ class LicenseTask extends DefaultTask {
     @SkipWhenEmpty
     FileCollection files
```

Comment:
If `getMatchingFiles` is the true input, shouldn't this be marked as `@Internal` now?

Reply:
This PR's scope is extremely basic: Remove the warnings generated by Gradle, though I guess I can take it a step further and optimize the tasks.